src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -160,23 +160,29 @@ internal abstract partial class DefinitionItem
             }
         }
 
-#pragma warning disable CS0612 // Type or member is obsolete - TypeScript
```

Comment:
The "TypeScript" comment here is getting lost which I presume means we had these obsolete due to IVT reasons? Want to restore the comment in some way? (Or is TypeScript off the IVT now?)

Reply:
i have restored the comment.  but i need to check in with TS soon as to their state here to see if they've moved off of this.